nodes/persistence-node/src/services/IpfsGatewayApi.ts

```diff
@@ -64,11 +72,16 @@ export class IpfsGatewayApi {
 
     app.get('/api/v0/cat', handleError(async (req, res) => {
       const hash = req.query.arg as string;
+      const filename = req.query.filename as string;
```

Comment:
this isn't a required query argument

Reply:
Should we make it required? If we omit it, file download just returns the file with file name `cat`.

Reply:
Fixed.